src/blockKit.js

```diff
@@ -134,48 +143,12 @@ module.exports.getOpenMessage = function({date, questionText, responses}) {
 				type: 'mrkdwn',
 				text: responsesMarkdown.join('\n')
 			}
-		})
+		});
 	}
 
 	return blocks;
 };
 
-// https://api.slack.com/tools/block-kit-builder?mode=message&blocks=%5B%7B%22type%22%3A%22section%22%2C%22text%22%3A%7B%22type%22%3A%22mrkdwn%22%2C%22text%22%3A%22Trivia%20question%20for%20November%208%2C%202019%5Cn%3E%20Harald%2C%20nicknamed%20Tangle-hair%2C%20took%20an%20oath%20not%20to%20cut%20or%20comb%20his%20hair%20until%20he%20became%20the%20first%20king%20of%20a%20united%20Norway.%5Cn*Answer%3A*%20Fact%5Cn%3E%20After%20achieving%20his%20goal%20ten%20years%20later%2C%20Harald%20cut%20his%20hair%2C%20trimmed%20his%20beard%2C%20and%20became%20known%20as%20Harald%20Fairhair%20or%20Finehair%2C%20King%20of%20Norway.%20%5C%22Hair%5C%22%20to%20the%20throne!%22%7D%7D%2C%7B%22type%22%3A%22section%22%2C%22text%22%3A%7B%22type%22%3A%22mrkdwn%22%2C%22text%22%3A%22%3Aheavy_check_mark%3A%20%3A%2B1%3A%20Alice%5Cn%3Ax%3A%20%3A-1%3A%20Bob%5Cn%3Aheavy_check_mark%3A%20%3A%2B1%3A%20Carolyn%5Cn%3Aheavy_check_mark%3A%20%3A%2B1%3A%20Dave%22%7D%7D%5D
-module.exports.getClosedMessage = function({date, questionText, answer, answerText, responses}) {
-	let mainMarkdown = `Trivia question for ${dateFormat(date, 'mmmm d, yyyy')}`;
-	if (questionText) {
-		mainMarkdown += `\n> ${questionText}`;
-	}
-	mainMarkdown +=`\n*Answer:* ${answer ? 'Fact' : 'Crap'}`;
-	if (answerText) {
-		mainMarkdown += `\n> ${answerText}`;
-	}
-
-	const responsesMarkdown = [];
-	responses.forEach(resp => {
-		const answerEmoji = (resp.answer === answer) ? ':heavy_check_mark:' : ':x:';
```

Comment:
We lost this in the merge of the functions. When an answer for the question is specified we need to indicate whether everyone's vote was correct or not.